tests/main/snapd-reexec-prompt/task.yaml

```diff
@@ -0,0 +1,33 @@
+summary: Test that snapd prompt services reexec into the snapd snap
+
+# Disable for Fedora, openSUSE and Arch as re-exec is not support there yet
+systems: [-ubuntu-core-*, -fedora-*, -opensuse-*, -arch-*, -amazon-*, -centos-*]
```

Comment:
what should happen with these services on core?  ATM I think on UC18+ they get installed by snapd I think

Reply:
Updated: I enabled core, but I excluded 14.04 (I guess the kernel bits won't land there).